### DISCUSSION TITLE
Any potential updates to the PKMDS app?

### DISCUSSION THREAD

**Discussion by Krimdra** [codemonkey85/PKMDS-Blazor#40] [Q&A] [answered]
I recently stumbled upon this app while looking for a way to edit my pokemon save files without pkhex (I currently don't have a PC and for now am using my Android Tablet). While I have found the app useful, it seems to be very limited compared to the full pkhex. I was wondering if there were any plans to update the app in the future.

**Comment by codemonkey85** [comment, marked as answer]
There are plans to try and make this app as capable as PKHeX, but this is a hobby project, and finding time to actually work on it is not easy.

**Reply by Krimdra** [reply]
![3upwmq](https://github.com/user-attachments/assets/276e6fc9-b755-4dee-bfa3-b80e2db81fb1)

**Comment by codemonkey85** [comment]
@Krimdra FYI, I've done a lot of work the last few weeks. Feel free to let me know if anything specific / important is missing. 😁

**Reply by Krimdra** [reply]
An "import pokemon file" feature would be great but otherwise looks good. Keep up the good work 👌🏾
